packages/react/src/link/Link.tsx

```diff
@@ -1,8 +1,11 @@
 import { Link as ChakraLink, LinkProps as ChakraLinkProps, forwardRef } from "@chakra-ui/react";
 
 export type LinkProps = Omit<ChakraLinkProps, "variant"> & {
-  variant?: "primary" | "secondary" | "tertiary";
+  /** The colorvariant of the link.
+   */
+  variant: "green" | "blue";
```

Comment:
Tror det vil være bedre å bruke colorScheme i stedet for variant her, siden det tross alt bare er fargene som endres.

Reply:
forsåvidt enig at det er mer intuitivt; men designere har fulgt dette fra Chakra, og det defineStyleConfig som benyttes for å style tillatter ikke colorScheme som parameter (bare som defaultprop). tar bare: baseStyle, sizes, variants. Tror ikke  det går å overkjøre det på noen vis...

Reply:
Hadde det ikke virket å gjøre det samme som på f.eks. button 
![image](https://github.com/kartverket/kvib/assets/122277136/2b873bd3-daa0-46c5-ad16-de5aa35e4ce0)

Reply:
Jeg har kikket litt på dette, og har laget et kodeeksempel som viser hvordan det vil se ut med å bruke colorScheme: https://github.com/kartverket/kvib/pull/285/files

Reply:
endret alt slik at det nå bruker colorscheme istedenfor variant; bedre løsning